openslides/users/views.py

```diff
@@ -110,11 +111,22 @@ def check_permissions(self, request):
         permission to see users and in case of create, update or destroy
         requests the permission to see extra user data and to manage users.
         """
-        if (not request.user.has_perm('users.can_see_name') or
-                (self.action in ('create', 'update', 'destroy') and not
-                 (request.user.has_perm('users.can_manage') and
-                  request.user.has_perm('users.can_see_extra_data')))):
-            self.permission_denied(request)
+        # Any logged in user can retrive groups.
```

Comment:
Hm. I don't like this style (using multiple returns in a difficult function). We can merge this for now but I would like to refactor all permission checks later. May be we should use explicit permission checks in the view methods instead of the general method `check_permission`.